From 0a93a680c6b7edf2a495a48e7a18c5e10433b118 Mon Sep 17 00:00:00 2001
From: Amelie Delaunay <amelie.delaunay@foss.st.com>
Date: Fri, 29 Mar 2024 11:10:23 +0100
Subject: [PATCH 0629/1141] arm64: dts: st: avoid scmi regulator yaml warning
 for stm32mp257f-ev1

Avoid an SCMI YAML code checking error.
The SCMI YAML oblige the node name to be regulator@xx.

Change-Id: I476a857f7631960d6569aef38c81a2dbd40f81dc
Signed-off-by: Pascal Paillet <p.paillet@st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/338481
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Domain-Review: Patrick DELAUNAY <patrick.delaunay@foss.st.com>
Reviewed-by: Patrick DELAUNAY <patrick.delaunay@foss.st.com>
---
 arch/arm64/boot/dts/st/stm32mp257f-ev1.dts | 14 +++++++-------
 1 file changed, 7 insertions(+), 7 deletions(-)

--- a/arch/arm64/boot/dts/st/stm32mp257f-ev1.dts
+++ b/arch/arm64/boot/dts/st/stm32mp257f-ev1.dts
@@ -532,22 +532,22 @@
 };
 
 &scmi_regu {
-	scmi_v3v3: voltd-v3v3 {
+	scmi_vddio1: regulator@0 {
+		regulator-min-microvolt = <1800000>;
+		regulator-max-microvolt = <3300000>;
+	};
+	scmi_v3v3: regulator@16 {
 		reg = <VOLTD_SCMI_STPMIC2_BUCK7>;
 		regulator-name = "v3v3";
 	};
-	scmi_vdd_emmc: voltd-vdd-emmc {
+	scmi_vdd_emmc: regulator@18 {
 		reg = <VOLTD_SCMI_STPMIC2_LDO2>;
 		regulator-name = "vdd_emmc";
 	};
-	scmi_vdd_sdcard: voltd-vdd-sdcard {
+	scmi_vdd_sdcard: regulator@23 {
 		reg = <VOLTD_SCMI_STPMIC2_LDO7>;
 		regulator-name = "vdd_sdcard";
 	};
-	scmi_vddio1: voltd-vddio1 {
-		regulator-min-microvolt = <1800000>;
-		regulator-max-microvolt = <3300000>;
-	};
 };
 
 &sdmmc1 {
